.eslintrc.json

```diff
@@ -3,6 +3,9 @@
   "env": {
     "browser": true
   },
+  "extends": [
+    "eslint:recommended"
+  ],
   "globals": {
     "DocumentTouch": true
```

Comment:
makes me wonder if the check is proper since `DocumentTouch` is not defined?

Reply:
I believe it's needed because of [the code you pointed out below](https://github.com/bryanbraun/anchorjs/pull/154#pullrequestreview-496999725).

Reply:
I know, but makes me wonder if the related code is correct since `DocumentTouch` isn't part of the `window` object. :)

Reply:
Yeah it looks like [the feature was deprecated](https://developer.mozilla.org/en-US/docs/Web/API/DocumentTouch). I'm guessing it only remained in Modernizr's code for compatibility purposes.